docs/UserGuide.md

```diff
@@ -22,17 +22,19 @@ guest management tasks done faster than traditional GUI apps.
 
```

Comment:
For the above, should we include the table of contents? Or should we wait till later to fill it up?

Reply:
I think we can fill it up after our meeting with everyone!